edge-apps/clock/screenly.yml

```diff
@@ -0,0 +1,42 @@
+app_id: ''
+root_asset_id: ''
+user_version: 'v1.0.0'
+revision: 1
+description: 'Screenly Clock Edge App'
+icon: ''
+author: 'Screenly, Inc.'
+homepage_url: ''
+settings:
+  override_timezone:
+    type: text
+    default: "false"
```

Comment:
Please double-check the cli... There is no default field. It's default_value.
There is no title field. Well, there is but title is used from dict name above. It is "override_timezone" so remove the title...